pydicom/config.py

```diff
@@ -152,15 +153,96 @@ def DS_decimal(use_Decimal_boolean: bool = True) -> None:
 Default ``False``.
 """
 
+
 enforce_valid_values = False
-"""Raise exceptions if any value is not allowed by DICOM Standard.
+"""Obsolete.
+Use :attr:`Settings.reading_validation_mode` instead.
```

Comment:
Sure, "deprecated" is what I actually meant - just my usual struggle with wording... Also `__getattr__` makes sense to warn at least for the getters (as I mentioned, this does not work for Python 3.6, but this will be EOL in a few days anyway), though I see no way to do this for the setters, too.

Reply:
Agree.  I checked before I made my comment - it still does not appear possible to do module-level setters in any Python version yet.

Reply:
At a second thought, it probably doesn't really help to warn on the getters, as a user would not use them, just the setters. The getters are only used by our own code.

Reply:
You are right - perhaps it can be warned in a generic upward-compatibility direction through a function like I list in my latest comment below?  Perhaps then it can go away in pydicom code, but still be triggered in user code.